base/special/trig.jl

```diff
@@ -102,22 +103,21 @@ function cos(x::T) where T<:Union{Float32, Float64}
             return T(1.0)
         end
         return cos_kernel(x)
-    elseif isnan(x)
-        return T(NaN)
-    elseif isinf(x)
+    end
+    if !isfinite(x)
+        isnan(x) && return x
         cos_domain_error(x)
+    end
+    n, y = rem_pio2_kernel(x)
+    n = n&3
+    if n == 0
+	return cos_kernel(y)
```

Comment:
format.

Reply:
?? What would you like changed?

Reply:
the returns in the `if` should be indented.

Reply:
The source file is correctly indented. I'm not sure how Git is confusing things.

Reply:
ah, it's tabs vs spaces.

Reply:
See also the relevant section in [CONTRIBUTING.md](https://github.com/JuliaLang/julia/blob/master/CONTRIBUTING.md#code-formatting-guidelines).

Reply:
Oops, sorry. I know better. New laptop; forgot to fix Emacs settings.

Fixed.